Applications/Spire/Source/Ui/Button.cpp

```diff
@@ -37,6 +37,8 @@ void Button::focusOutEvent(QFocusEvent* event) {
   if(event->reason() != Qt::PopupFocusReason && m_is_down) {
     m_is_down = false;
   }
+  unmatch(*this, Press());
+  unmatch(*this, Hover());
```

Comment:
What does line 41 correspond to in the spec?

Reply:
It doesn't; this should be moved to the Box.